index.html

```diff
@@ -25,7 +26,60 @@ <h2 class="hero-heading">This is solid state </h2>
+            <section class="blog">
+                <article class="article container">
+                    <header>
+                        <div>
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="photo.jpg" alt="Camera">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!</p>
+                       <a class="article-link" href="">Learn More</a>
+                </article>
+            </section>
+            <section class="blog blog-two">
+                <article class="article container">
+                    <header>
+                        <div>
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="images.jpg" alt="Lens">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!</p>
+                       <a class="article-link" href="">Learn More</a>
+                </article>
+            </section>
+            <section class="blog blog-third">
+                <article class="article container">
+                    <header>
+                        <div>
+                          <h3 class="third-header">Magna Arcu Feugiat</h3>
+                          <img src="bug.jpg.crdownload" alt="Bug">
+                        </div>
+                    </header>
+                    <hr>
+                    <p class="article-text">Lorem, ipsum dolor sit amet consectetur adipisicing Lorem ipsum dolor sitr adipisicing elit.
+                       Repudiandae nihil aut quisquam nostrum exercitationem voluptatem neque necessitatibus fugiat! Esse error aliquid beatae.
+                       Dolore quaerat, in ipsam numquam laudantium doloribus debitis,
+                       corporis aspernatur odit officiis ut blanditiis ex hic dolores fugit quisquam veniam recusandae placeat!</p>
+                       <a class="article-link" href="#">Learn More</a>
```

Comment:
Same here, please make sure the closing `p` is exactly below the opening `p` and place the `a` element below the `p`.